tests/test_density_RDF.py

```diff
@@ -13,25 +13,23 @@
 
 
 class TestRDF:
-    def test_generateR(self):
+    @pytest.mark.parametrize("r_min", [0, 0.05, 0.1, 1.0, 3.0])
+    def test_generateR(self, r_min):
```

Comment:
It might make sense to rename this test to `test_generate_bins`.
```suggestion
    def test_generate_bins(self, r_min):
```